medium-tests/src/test/java/mediumtest/analysis/AnalysisTriggeringMediumTests.java

```diff
@@ -221,4 +225,42 @@ void it_should_analyze_open_files_when_enabling_rule(@TempDir Path baseDir) {
         .extracting(RaisedIssueDto::getPrimaryMessage)
         .containsExactly("Update this \"artifactId\" to match the provided regular expression: '[a-z][a-z-0-9]+'"));
   }
+
+  @Test
```

Comment:
I think at least one more test is needed to demonstrate that analysis triggering is happening if some rules are enabled.

Reply:
[The existing test](https://github.com/SonarSource/sonarlint-core/blob/aae10754e4e6de46c1749acf21deb87a1c1a323f/medium-tests/src/test/java/mediumtest/analysis/AnalysisTriggeringMediumTests.java#L197) covers this case, no?